lib/mongodb.js

```diff
@@ -67,7 +67,11 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
   dataSource.ObjectID = mongodb.ObjectID;
 
   if (callback) {
-    dataSource.connector.connect(callback);
+    if (s.lazyConnect) {
+      callback(null);
```

Comment:
Thank you @raymondfeng , applied. 
hmm, is there any reason or benefit to callback after the return? Since it's already the last step in function `initialize`